includes/querypages/QueryPage.php

```diff
@@ -247,4 +247,11 @@ function doQuery( $offset = false, $limit = false, $property = false ) {
 
 		return $num;
 	}
+
+	/**
+	 * @return array|bool
+	 */
+	public function getQueryInfo() {
+		return null;
```

Comment:
Does not match the type doc. If the real type is array|bool, then you can do

```php
public function getQueryInfo(): ?array {
```

Reply:
Thanks, done.

Reply:
Why not the PHP type instead of the comment?

Reply:
Oh, I didn’t realise you wanted that. Done now. Thanks!